elastic/integrations
# Add synthetics estimator dashboard based on adhoc data view

## What does this PR do?

Adds synthetics estimator dashboard.

## Checklist


## How to test this PR locally

1. Build package locally:
````
// from integrations/packages/synthetics
❯ elastic-package build
2023/06/29 15:33:23  WARN CommitHash is undefined, in both //.elastic-package/version and the compiled binary, config may be out of date.
Build the package
2023/06/29 15:33:23  INFO License text found in "/repos/integrations/LICENSE.txt" will be included in package
Package built: <pwd>/integrations/build/packages/synthetics-1.0.2.zip
Done
````

2. Set up environment variables to install the package remotely into an existing cluster:
````
integrations/packages/synthetics (225af28) [!?]
❯ export ELASTIC_PACKAGE_ELASTICSEARCH_USERNAME="elastic"

integrations/packages/synthetics (225af28) [!?]
❯ export ELASTIC_PACKAGE_ELASTICSEARCH_PASSWORD="<password>"

integrations/packages/synthetics (225af28) [!?]
❯ export ELASTIC_PACKAGE_KIBANA_HOST="<url>"
````
3. Install the package:
````
❯ elastic-package install --zip ./integrations/build/packages/synthetics-1.0.2.zip -v
2023/06/29 15:18:28  WARN CommitHash is undefined, in both /.elastic-package/version and the compiled binary, config may be out of date.
2023/06/29 15:18:28 DEBUG Enable verbose logging
2023/06/29 15:18:29 DEBUG GET <url>.io:9243/api/status
2023/06/29 15:18:29 DEBUG Validating built .zip package (path: /repos/integrations/build/packages/synthetics-1.0.2.zip)
2023/06/29 15:18:29 DEBUG Skip validation of the built .zip package
2023/06/29 15:18:29 DEBUG POST <url>.io:9243/api/fleet/epm/packages
```` 
4. Log into Kibana and check installed version matches:
![image](https://github.com/elastic/integrations/assets/95703246/c599a929-e2e0-4aa2-903f-9b0f0019de3a)

5. Open synthetics dashboard and check data is displayed:
![image](https://github.com/elastic/integrations/assets/95703246/dfca22f2-62e4-4aad-a11b-f235558a0fb0)
